web/src/components/tasks/search-business-name/SearchBusinessNameForm.test.tsx

```diff
@@ -140,7 +140,7 @@ describe("<SearchBusinessNameForm />", () => {
   it("shows restricted word text if name includes a restricted word", async () => {
```

Comment:
You could do the same here so the naming is the same, but I think it's okay not to hear, b/c I think `restricted word` more intuitively indicates that there will be an error or, at the least, some sort of barrier.